test/system/010-images.bats

```diff
@@ -359,4 +359,41 @@ EOF
     run_podman --root $imstore/root rmi --all
 }
 
+@test "podman images with concurrent removal" {
+    skip_if_remote "following test is not supported for remote clients"
+    local count=5
+
+    # First build $count images
+    for i in $(seq --format '%02g' 1 $count); do
+        cat >$PODMAN_TMPDIR/Containerfile <<EOF
+FROM $IMAGE
+RUN echo $i
+EOF
+        run_podman build -q -t i$i $PODMAN_TMPDIR
+    done
+
+    run_podman images
+    # Now remove all images in parallel and in the background and make sure
+    # that listing all images does not fail (see BZ 2216700).
+    for i in $(seq --format '%02g' 1 $count); do
+        timeout --foreground -v --kill=10 60 \
+                $PODMAN rmi i$i &
+    done
+
+    tries=100
+    while [[ ${#lines[*]} -gt 1 ]] && [[ $tries -gt 0 ]]; do
+        # Prior to #18980, 'podman images' during rmi could fail with 'image not known'
+        run_podman images --format Name
```

Comment:
I'm sorry, I wrote an abbreviated terse comment that was not actual code. I did not mean literal `--format Name`, because that is nonsensical. I just didn't want to type the curly braces dots quotes because I was in a hurry. I'm sorry.

Please change this to:
```bash
run_podman images --format '{{.ID}} {{.Names}}'
```
Reason: if the timeout test fails below, having both IID and image name will make it easier for a maintainer to understand what went wrong.

With this change, you can remove line 391 (`run_podman images`) below, because it does not contribute anything useful. The useful part of the error is "podman images spits out more than one line, and I was expecting only one line". The person investigating a test failure can easily see that already from all the repeated `podman images` calls in this loop.

I hope this makes sense. I am sorry for writing in shortcuts.

Reply:
Alternately, `run_podman images --noheading` might be even cleaner and easier for a maintainer to understand.